torch/distributed/tensor/experimental/_attention.py

```diff
@@ -1291,3 +1291,22 @@ def context_parallel_unshard(
         else _SequentialSharder
     )
     return [sharder.unshard(b, mesh, dim) for b, dim in zip(buffers, seq_dims)]
+
+
+def set_rotate_method(rotate_method: str) -> None:
+    """
+    Context Parallel SDPA requires the rotation of kv shards. Users can call this
+    API to specify which rotation method to use. "alltoall" shuffles the kv shards
+    using all-to-all collective while "allgather" gathers the kv shards using
+    all-gather collective. If this API has not been called, the default rotate
+    method is "allgather".
+    """
```

Comment:
Can we also add

```
Args:
    rotate_method(str): ...
```

I notice that I didn't do this for `context_parallel_unshard` either. We may want to fix it later. Sorry about the bad documentation.